### PR TITLE
TDEAL-2: Add EnableAutoSetupIfNotUITesting extension method

### PR DESCRIPTION
**Pull request by sarahelsaig**
[TDEAL-2](https://lombiq.atlassian.net/browse/TDEAL-2)

[TDEAL-2]: https://lombiq.atlassian.net/browse/TDEAL-2?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Piedone — CHANGES_REQUESTED** [review]
Merge, please.

**Comment by sarahelsaig** [comment]
atlassian/gajira-transition has some API error. Is this related to OSOE-726? It's definitely not related to this PR.

**Comment by Piedone** [comment]
These actions should continue to work still, so I don't know. The error is also pretty useless.

**Comment by Piedone** [comment]
Ah I figured out, there was some missing auth config for the project, fixed it now. Will work from now on.

File: src/Lombiq.OSOCE.Web/Program.cs [hunk 1]
```diff
@@ -16,13 +16,14 @@
...
     .AddOrchardCms(orchardCoreBuilder => orchardCoreBuilder
+        .AddOrchardCoreApplicationInsightsTelemetry(configuration)
         .ConfigureFeaturesGuard(
             new Dictionary<string, IEnumerable<string>>
             {
                 ["OrchardCore.Twitter"] = new[]
                 {
-                    Lombiq.UIKit.FeatureIds.Base,
-                    Lombiq.ChartJs.Constants.FeatureIds.Default,
+                    "Lombiq.UIKit",
```
Comment (Piedone) [inline src/Lombiq.OSOCE.Web/Program.cs]: Why go back to the strings from the consts?

**Review by Piedone — CHANGES_REQUESTED** [review]

File: src/Lombiq.OSOCE.Web/Program.cs (hunk 1 above)
Comment (sarahelsaig) [reply src/Lombiq.OSOCE.Web/Program.cs]: Messed up merge conflict. :/

**Review by Piedone — APPROVED** [review]

**Pull request merged**